pkg/cvo/internal/operatorstatus.go

```diff
@@ -203,14 +203,20 @@ func checkOperatorHealth(ctx context.Context, client ClusterOperatorsGetter, exp
 		}
 	}
 
-	// during initialization we allow degraded
-	if degraded && mode != resourcebuilder.InitializingMode {
+	if degraded {
 		if degradedCondition != nil && len(degradedCondition.Message) > 0 {
 			nestedMessage = fmt.Errorf("cluster operator %s is %s=%s: %s, %s", actual.Name, degradedCondition.Type, degradedCondition.Status, degradedCondition.Reason, degradedCondition.Message)
 		}
+		var updateEffect payload.UpdateEffectType
+
+		if mode == resourcebuilder.InitializingMode {
+			updateEffect = payload.UpdateEffectReport
```

Comment:
In order to make this non-blocking (which we want), I think we'll also need code around [here][1] like:

```go
if err := task.Run(ctx, payloadUpdate.Release.Version, w.builder, work.State); err != nil {
  if uErr, ok := err.(*payload.UpdateError); ok && uErr.UpdateEffect == payload.UpdateEffectReport {
    // do not fail the task on this manifest, just record it for later complaining
    reportEffectErrors = append(reportEffectErrors, err)
  } else {
    return err
  }
}
```

And then down around [here][2], something like:

```go
// update the status
cr.Complete()
return errors.NewAggregate(reportEffectErrors)
```

[1]: https://github.com/openshift/cluster-version-operator/blob/d3f4ca856badfe3db7a74415bd4a418d128999b1/pkg/cvo/sync_worker.go#L762
[2]: https://github.com/openshift/cluster-version-operator/blob/d3f4ca856badfe3db7a74415bd4a418d128999b1/pkg/cvo/sync_worker.go#L778